controls/hipaa.yml

```diff
@@ -117,7 +117,11 @@ controls:
             - audit_rules_privileged_commands_unix_chkpwd
             - audit_rules_privileged_commands_userhelper
             - audit_rules_immutable
+            {{% if product == "rhel10" %}}
+            - audit_rules_mac_modification_etc_selinux
```

Comment:
The rule audit_rules_mac_modification is a part of multiple controls in this HIPAA control file. If you want to make sure this rule isn't present in the RHEL 10 data stream you need to take care of all occurrences of it in this control file.